examples/create-tables-with-query/create_tables_with_query.md

```diff
@@ -0,0 +1,19 @@
+# Create tables with query
+
+Query expressions can be used to create tables from an iterable value. The `table` keyword should be specified as the query construct type before the query expression. The key of the created table can be specified explicitly.
```

Comment:
we can create a table even without the `table` keyword if we are iterating a table itself.